src/elements/content-sidebar/activity-feed/comment/CommentDeleteConfirmation.js

```diff
@@ -4,22 +4,20 @@
  */
 
 import * as React from 'react';
-import { FormattedMessage } from 'react-intl';
+import { FormattedMessage, type MessageDescriptor } from 'react-intl';
 
-import { Overlay } from '../../../../components/flyout';
-import PrimaryButton from '../../../../components/primary-button';
 import Button from '../../../../components/button';
 import commonMessages from '../../../common/messages';
-import messages from './messages';
+import PrimaryButton from '../../../../components/primary-button';
+import { KEYS } from '../../../../constants';
+import { Overlay } from '../../../../components/flyout';
 import './CommentDeleteConfirmation.scss';
 
-import { COMMENT_TYPE_DEFAULT, COMMENT_TYPE_TASK, KEYS } from '../../../../constants';
-
 type Props = {
     isOpen: boolean,
+    message: MessageDescriptor,
     onDeleteCancel: Function,
     onDeleteConfirm: Function,
-    type: typeof COMMENT_TYPE_DEFAULT | typeof COMMENT_TYPE_TASK,
 };
 
 class CommentDeleteConfirmation extends React.Component<Props> {
```

Comment:
Sateless, maybe make it at least PureComponent for now?

Reply:
Maybe, but I don't plan to refactor this component further as part of this pull request.